docs/istio-on-karmada.md

```diff
@@ -0,0 +1,457 @@
+# Use Istio on Karmada
+
+This document uses an example to demonstrate how to use [Istio](https://istio.io/) on Karmada.
+
+Follow this guide to install the Istio control plane on karmada-host (the primary cluster) and configure member1 and member2 (the remote cluster) to use the control plane in karmada-host. All clusters reside on the network1 network, meaning there is direct connectivity between the pods in both clusters.
+
+<image src="images/istio-on-karmada.png"
+    caption="Istio on Karmada"
+    />
+
+## Install Karmada
+
+### Install karmada control plane
+
+Following the steps [Install karmada control plane](https://github.com/karmada-io/karmada#install-karmada-control-plane) in Quick Start, you can get a Karmada. 
+
+## Deploy Istio
+
+***
+If you are testing multicluster setup on `kind` you can use [MetalLB](https://metallb.universe.tf/installation/) to make use of `EXTERNAL-IP` for `LoadBalancer` services.
+***
+
+### Install istioctl
+Please refer to the [istioctl](https://istio.io/latest/docs/setup/getting-started/#download) Installation.
+
+### Prepare cacerts
+
+Following the steps [plugin-ca-cert](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/) to create a secret cacert.
+```bash
+root@karmada-demo  istio-on-karmada# tree certs
+certs
+├── primary
+│   ├── ca-cert.pem
+│   ├── ca-key.pem
+│   ├── cert-chain.pem
+│   └── root-cert.pem
+├── root-ca.conf
+├── root-cert.csr
+├── root-cert.pem
+├── root-cert.srl
+└── root-key.pem
+```
+### Install Istio on karmada-apiserver
+
+Export `KUBECONFIG` and switch to `karmada apiserver`:
+
+```
+# export KUBECONFIG=$HOME/.kube/karmada.config
+
+# kubectl config use-context karmada-apiserver 
+```
+
+Create a secret `cacerts` in `istio-system` namespace:
+```bash
+kubectl create namespace istio-system
+kubectl create secret generic cacerts -n istio-system \
+    --from-file=certs/primary/ca-cert.pem \
+    --from-file=certs/primary/ca-key.pem \
+    --from-file=certs/primary/root-cert.pem \
+    --from-file=certs/primary/cert-chain.pem
+```
+
+Create a  propagation policy for `cacert` secret:
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: cacerts-propagation
+  namespace: istio-system
+spec:
+  resourceSelectors:
+    - apiVersion: v1
+      kind: Secret
+      name: cacerts
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+EOF
+```
+
+Run the following command to install istio CRDs on karmada apiserver: 
+```bash
+cat <<EOF | istioctl install -y --set profile=minimal -f -
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogFile: /dev/stdout
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: primary
+      network: network1
+EOF
+```
+karma apiserver will not deploy a real istiod pod, you should press `ctrl+c` to exit installation.
```

Comment:
s/karma/Karmada

Reply:
Maybe you need to show what has been installed, and let users know when it is successful